src/app/sb-admin/components/pages/sb-user/sbuser.component.ts

```diff
@@ -147,5 +146,30 @@ export class SbUserComponent implements OnInit {
             height: 'auto'
         });
     }
-}
 
+  blockUnblockUser(user: User) {
+    this.blockUnblockUserDialog = true;
+    this.user = user;
+  }
+  confirmBlock() {
+    const payload = {
+      "request": {
+        "userId": this.user.userId
+      }
+    }
+    this.userService.blockUnblockUser(payload, this.user?.status).subscribe(response => {
+      this.messages = [];
```

Comment:
what is the use of this statement?

Reply:
empty the previous message and show the current one